README.md

```diff
@@ -57,6 +57,7 @@ Per-VDOM:
  * `fortigate_ipsec_tunnel_receive_bytes_total`
  * `fortigate_ipsec_tunnel_transmit_bytes_total`
  * `fortigate_ipsec_tunnel_up`
+ * `fortigate_bgp_neighbors`
```

Comment:
Please rename the metrics to `fortigate_bgp_neighbor_ipv4_info` and  `fortigate_bgp_neighbor_ipv6_info`   
As `_info` signals that this is an info metric and contains the information in the labels and use singular as the information is about a single bgp neighbor.

Reply:
Agreed, fixed :)